sysutils/ftp-backup/pkg-descr

```diff
@@ -0,0 +1,4 @@
+This package adds a backup option using a FTP server.
+
+Due to the sensitive nature of the data being send to the backup, we
+strongly advise to not use a public service to send backups to.
```

Comment:
I would also mention the optional usage of TLS (at least on the data connection).